pep-0012.rst

```diff
@@ -60,14 +60,12 @@ Once you've decided which type of PEP yours is going to be, follow the
 directions below.
 
 - Make a copy of this file (the ``.rst`` file, **not** the HTML!) and
-  perform the following edits. Name the new file ``pep-9999.rst`` if
-  you don't yet have a PEP number assignment, or ``pep-NNNN.rst`` if
-  you do. Those with push permissions are welcome to claim the next
-  available number (ignoring the special blocks 3000 and 8000, and a
-  handful of special allocations - currently 666, 754, and 801) and
-  push it directly.
-
-- Replace the "PEP: 12" header with "PEP: 9999" or "PEP: NNNN",
+  perform the following edits. Name the new file ``pep-NNNN.rst``, using
+  the next available number (not used by a published or in-PR PEP, ignoring
+  the special blocks 3000 and 8000, and a handful of special allocations -
+  currently 666, 754, and 801).
```

Comment:
```suggestion
  the next available number (not used by a published or in-PR PEP).
```

IMO, we can also eliminate the explicit references to these three historical special-case numbers here, as it just adds extra complexity for the reader and maintenance burden for us when they are just published PEPs like any other that an author would see in the same manner as other PEPs in the sequential index, and one of the numbers has already been passed by the normal PEP sequence. Likewise, as it will likely be at least several decades until we reach the existing reserved number ranges (if we ever do), there's no point bothering authors with this now.